packages/smart/src/deployer.ts

```diff
@@ -200,9 +181,9 @@ export class TurboCreator {
     specified: Partial<TrustedArbiterConfiguration> & { arbiter: string }
   ): TrustedArbiterConfiguration {
     return {
-      creatorFee: ethers.BigNumber.from(10).pow(16),
+      creatorFee: BigNumber.from(10).pow(16),
       outcomeSymbols: ["NO", "YES"],
-      outcomeNames: ["NO", "YES"].map(ethers.utils.formatBytes32String),
+      outcomeNames: ["NO", "YES"].map(utils.formatBytes32String),
```

Comment:
Need to start using outcome name constants so we can be consistent across the repo

Reply:
good point